Cilsil/Utils/ProgramState.cs

```diff
@@ -414,6 +414,18 @@ public void PushRetExpr()
             if (currentSequencePoint != null)
             {
                 var newLocation = Location.FromSequencePoint(currentSequencePoint);
+                var previousInstruction = CurrentInstruction.Previous;
```

Comment:
This change fixes big line number found in CFG

Reply:
I think I know the issue you're referring to, but can you provide the before/after examples of what this looks like in practice?

Reply:
> I think I know the issue you're referring to, but can you provide the before/after examples of what this looks like in practice?

For the following code snippet:
```cs
StreamWriter stream = AllocateStreamWriter();
if (stream == null)
    return;
```
we used to have a branch node like this:
```
node71 preds: 70 succs: 72 exn: 69 Call
_=*n$0:Examples.Program [Line 99];
n$1:System.IO.StreamWriter*=_fun System.IO.StreamWriter Examples.Program.AllocateStreamWriter() (n$0:Examples.Program*) [Line 99];
*&%0:System.IO.StreamWriter*=n$1 [Line 99];

node72 preds: 71 succs: 73 74 exn: 69 MethodBody
n$3=*&%0:System.IO.StreamWriter* [Line 100];

node73 preds: 72 succs: 68 exn: 69 MethodBody
PRUNE (n$3 == null, True) **[Line 16707566]**;

node74 preds: 72 succs: 75 exn: 69 MethodBody
PRUNE (!(n$3 == null):, False) **[Line 16707566]**;
n$4=*&%0:System.IO.StreamWriter* [Line 105];
```

Now with the fix, the branch node is now like this:
```
node71 preds: 70 succs: 72 exn: 69 Call
_=*n$0:Examples.Program [Line 99];
n$1:System.IO.StreamWriter*=_fun System.IO.StreamWriter Examples.Program.AllocateStreamWriter() (n$0:Examples.Program*) [Line 99];
*&%0:System.IO.StreamWriter*=n$1 [Line 99];

node72 preds: 71 succs: 73 74 exn: 69 MethodBody
n$3=*&%0:System.IO.StreamWriter* [Line 100];

node73 preds: 72 succs: 68 exn: 69 MethodBody
PRUNE (n$3 == null, True) **[Line 100]**;

node74 preds: 72 succs: 75 exn: 69 MethodBody
PRUNE (!(n$3 == null):, False) **[Line 100]**;
n$4=*&%0:System.IO.StreamWriter* [Line 105];
```